Add external integrations section (#4)
open-cluster-management-io/ocm: README.md

* https://github.com/open-cluster-management-io/governance-policy-template-sync
* https://github.com/open-cluster-management-io/governance-policy-propagator

### More external integrations

We are constantly working with other open source projects to make multicluster management easier.

> - [Submariner](https://submariner.io/) is a project to provide multicluster networking connectivity. Users can benefit from a [Submariner](https://submariner.io/) addon, which automates the deployment and management of multi-cluster networking.

Reviewer: ```suggestion
- [Submariner](https://submariner.io/) is a project that provides multicluster networking connectivity. Users can benefit from a [Submariner](https://submariner.io/) addon, which automates the deployment and management of multicluster networking.
```
Small nit for consistency of `multicluster` word usage.

> - [Clusternet](http://github.com/clusternet/clusternet) is another project to provide multicluster orchestration, which can be easily plug into OCM with [clusternet addon](https://github.com/skeeey/clusternet-addon)

Reviewer: ```suggestion
- [Clusternet](http://github.com/clusternet/clusternet) is another project that provides multicluster orchestration, which can be easily plug into OCM with [clusternet addon](https://github.com/skeeey/clusternet-addon).
```
small nit comment.
